enterprise/messages.py

```diff
@@ -136,8 +113,10 @@ def add_generic_error_message_with_code(request, error_code):
             '{span_start}Please reach out to your learning administrator with '
             'the following error code and they will be able to help you out.{span_end}'
             '{span_start}Error code: {error_code}{span_end}'
+            '{span_start}Username: {username}{span_end}'
         ).format(
             error_code=error_code,
+            username=request.user.username,
```

Comment:
is there any case in which we are rendering this message without a logged-in user? If yes then you need to handle the edge case.

Reply:
1. We are using this method for authenticated users.
2. assuming that in the future there might be some scenario for unauthenticated users as well,  `request.user` will be an Anonymous user and anonymous_user. username will be an empty string.